Sqlx-core error in logger.rs FormatOptions reference

### Bug Description
When using `cargo install sea-orm-cli` for versions 1.0.0 and 1.0.1 I get an error for the sqlx-core crate as follows:

```bash
       error[E0308]: mismatched types
       --> /home/server/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqlx-core-0.7.4/src/logger.rs:112:29
       |
109 |                         sqlformat::format(
       |                         ----------------- arguments to this function are incorrect
...
112 |                             sqlformat::FormatOptions::default()
       |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `&FormatOptions<'_>`, found `FormatOptions<'_>`
       |
       note: function defined here
           --> /home/server/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqlformat-0.2.7/src/lib.rs:18:8
       |
18   | pub fn format(query: &str, params: &QueryParams, options: &FormatOptions) -> String {
       |        ^^^^^^
       help: consider borrowing here
       |
112 |                             &sqlformat::FormatOptions::default()
       |                             +

For more information about this error, try `rustc --explain E0308`.
error: could not compile `sqlx-core` (lib) due to 1 previous error
```

### Minimal Reproduction
Run `cargo install sea-orm-cli`

### Info
* SQLx version: v0.7.4
* SQLx features enabled: N/A
* Database server and version: N/A
* Operating system: Debian 12
* `rustc --version`: rustc 1.81.0 (eeb90cda1 2024-09-04)

--- a/sqlx-core/Cargo.toml
+++ b/sqlx-core/Cargo.toml
@@ -69,7 +69,7 @@ regex = { version = "1.5.5", optional = true }
 serde = { version = "1.0.132", features = ["derive", "rc"], optional = true }
 serde_json = { version = "1.0.73", features = ["raw_value"], optional = true }
 sha2 = { version = "0.10.0", default-features = false, optional = true }
-sqlformat = "0.2.0"
+#sqlformat = "0.2.0"
 thiserror = "1.0.30"
 tokio-stream = { version = "0.1.8", features = ["fs"], optional = true }
 tracing = { version = "0.1.37", features = ["log"] }

--- a/sqlx-core/src/logger.rs
+++ b/sqlx-core/src/logger.rs
@@ -60,8 +60,6 @@ pub(crate) fn private_level_filter_to_trace_level(
     private_level_filter_to_levels(filter).map(|(level, _)| level)
 }
 
-//pub use sqlformat;
-
 pub struct QueryLogger<'q> {
     sql: &'q str,
     rows_returned: u64,
@@ -112,13 +110,12 @@ impl<'q> QueryLogger<'q> {
                     summary.push_str(" …");
                     format!(
                         "\n\n{}\n",
-                        self.sql
-                        /*
-                        sqlformat::format(
-                            self.sql,
-                            &sqlformat::QueryParams::None,
-                            sqlformat::FormatOptions::default()
-                        )*/
+                        self.sql /*
+                                 sqlformat::format(
+                                     self.sql,
+                                     &sqlformat::QueryParams::None,
+                                     sqlformat::FormatOptions::default()
+                                 )*/
                     )
                 } else {
                     String::new()

--- a/sqlx-core/src/logger.rs
+++ b/sqlx-core/src/logger.rs
@@ -60,7 +60,7 @@ pub(crate) fn private_level_filter_to_trace_level(
     private_level_filter_to_levels(filter).map(|(level, _)| level)
 }
 
-pub use sqlformat;
+//pub use sqlformat;
 
 pub struct QueryLogger<'q> {
     sql: &'q str,
@@ -112,11 +112,13 @@ impl<'q> QueryLogger<'q> {
                     summary.push_str(" …");
                     format!(
                         "\n\n{}\n",
+                        self.sql
+                        /*
                         sqlformat::format(
                             self.sql,
                             &sqlformat::QueryParams::None,
                             sqlformat::FormatOptions::default()
-                        )
+                        )*/
                     )
                 } else {
                     String::new()

--- a/sqlx-sqlite/src/logger.rs
+++ b/sqlx-sqlite/src/logger.rs
@@ -416,11 +416,13 @@ impl<'q, R: Debug, S: Debug + DebugDiff, P: Debug> QueryPlanLogger<'q, R, S, P> 
             summary.push_str(" …");
             format!(
                 "\n\n{}\n",
-                sqlformat::format(
-                    self.sql,
-                    &sqlformat::QueryParams::None,
-                    sqlformat::FormatOptions::default()
-                )
+                self.sql /*
+                         sqlformat::format(
+                             self.sql,
+                             &sqlformat::QueryParams::None,
+                             sqlformat::FormatOptions::default()
+                         )
+                         */
             )
         } else {
             String::new()

--- a/src/ty_match.rs
+++ b/src/ty_match.rs
@@ -128,7 +128,7 @@ pub fn dupe_value<T>(_t: &T) -> T {
 
 #[test]
 fn test_dupe_value() {
-    let ref val = (String::new(),);
+    let val = &(String::new(),);
 
     if false {
         let _: i32 = dupe_value(&0i32);

